### ISSUE TITLE
clangd build fails

### ISSUE THREAD

**Issue by ldelossa** [electrickite/batsignal#22]
Normally would not care about this but I'm using an automated system to deploy laptop builds, it favors clang. 

```
make
/usr/bin/clang -pedantic -Wall -Wextra -Werror -Wno-unused-parameter -Os -std=c11 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -pthread  -O1   -c -o batsignal.o batsignal.c
batsignal.c:395:30: error: 'fscanf' may overflow; destination buffer in argument 3 has size 10, but the corresponding specifier may require size 11 [-Werror,-Wfortify-source]
    if (fscanf(file, "%10s", type) == 0) { /* Continue... */ }
```

Feel free to mark this as "wont fix" if you don't find clang support necessary.

**Comment by electrickite** [comment]
Thanks for the bug report! Resolved in version 1.6.1. Seems to build with clang now, but feel free to create a new issue if you run into any other problems.

**Comment by ldelossa** [comment]
thanks :)
